i18n_alchemy.gemspec

```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |s|
   s.test_files    = Dir["test/**/*"]
   s.require_paths = ["lib"]
 
-  s.add_dependency "activesupport", ">= 3.2.0", "<= 4.2.0"
+  s.add_dependency "activesupport", ">= 3.2.0", "<= 4.2.1"
```

Comment:
I think we should be more inclusive in the dependency range here or we won't support `4.2.2` (latest sec release) or `4.2.3` (the upcoming patch release).

So maybe `< 4.3.0` or `< 5.0.0`? /cc @carlosantoniodasilva